test/unit/middlewares/error-handler.spec.js

```diff
@@ -143,11 +143,11 @@
 			const ctx = Context.create(broker, eventEndpoint);
 
 			return newHandler(ctx)
-				.catch(protectReject)
-				.then(() => {
+				.then(() => { fail('error should be thrown to the transit message handler') })
```

Comment:
Codacy has a fix for the issue: [Replace `·fail('error·should·be·thrown·to·the·transit·message·handler')·` with `⏎↹↹↹↹↹fail("error·should·be·thrown·to·the·transit·message·handler");⏎↹↹↹↹`](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141835)
```suggestion
				.then(() => {
```